cmd/podman/search.go

```diff
@@ -345,6 +345,9 @@ func matchesOfficialFilter(filter searchFilterParams, result docker.SearchResult
 }
 
 func getRegistry(image string) (string, error) {
+	if image[len(image)-1] == '/' {
```

Comment:
Maybe use strings.HasSuffix and strings.TrimSuffix here to be absolutely clear what's going on?

Reply:
Agreed! Fixed